### ISSUE TITLE
Updated relationship between temperature & picpoc for coccos

### ISSUE THREAD

**Issue by kristenkrumhardt** [marbl-ecosys/MARBL#348]
In the compute_autotroph_calcification subroutine in marbl_interior_tendency_mod.F90, I have an updated relationship between temperature and coccolithophore PIC/POC production ratio. I've changed in from a linear relationship to a cubic one. It helps to better match PIC observations in the Southern Ocean particularly. 

This bit around line 1450 in marbl_interior_tendency_mod.F90:
```          !temperature effect (linearly decreases calcification at temps < 11C)
          where (temperature < 11._r8)
            picpoc = max(0._r8,0.104_r8 * temperature - 0.108_r8)
          elsewhere
            picpoc = 1._r8
          end where
```
should be changed to:

```          !temperature effect (new cubic function)
          where (temperature < 21.5_r8)
            picpoc = max(0._r8,0.0001_r8 * temperature**3)
          elsewhere
            picpoc = 1._r8
          end where
```

**Comment by mnlevy1981** [comment]
Should this be a continuous function? We could do that by replacing `21.5_r8` with the cube root of 10000 (21.54...) or replacing `0.0001_r8` with 1/21.5^3 (1.006...e-4). We'd want to use a parameter in either case, something like

```
real(r8), parameter :: picpoc_temp_thres = 10000._r8**(1._r8/3._r8)

...

          where (temperature < picpoc_temp_thres)
            picpoc = max(0._r8,0.0001_r8 * temperature**3)
          elsewhere
            picpoc = 1._r8
          end where
```

or

```
real(r8), parameter :: name_this_param = 1._r8/21.5_r8

...

          where (temperature < 21.5_r8)
            picpoc = max(0._r8, (name_this_param * temperature)**3)
          elsewhere
            picpoc = 1._r8
          end where
```

**Comment by kristenkrumhardt** [comment]
I think the first option would be good.

**Comment by mnlevy1981** [comment]
closed via #353, first available in [marbl0.39.0](https://github.com/marbl-ecosys/MARBL/releases/tag/marbl0.39.0)

For completeness, I'll mentioned that we used a third option for continuity as recommended by @klindsay28:

```
picpoc = min(1.0_r8, max(0.0_r8, 0.0001_r8 * temperature**3))
```
